### PR TITLE
Cart Cleanup

### PR DESCRIPTION
**Pull request by djoyal**
Cleaned up the CartContext, added `Next` button to `/Cart` page

(Sorry, forgot to include this in the last PR)

Screen Shot 2021-02-08 at 3 35 05 PM

### CODE REVIEW

**Review by jfplataroti — APPROVED** [review]
Nice cleanup!! 👍

**Comment by djoyal** [comment]
Thanks for the initial review! I added a bit more to the PR to cover some extra functionality

File: packages/web/components/cartItemSummary.tsx [hunk 1]
```diff
@@ -145,6 +149,32 @@ export default function CartItemSummaries() {
           setVisible={setShowQuantityEditor}
         />
       )}
+      {showConfirmationModal && (
+        <Layer full={false} modal responsive={false}>
+          <Card width="75vw">
```
Comment (jfplataroti) [inline packages/web/components/cartItemSummary.tsx:154]: nice `width="75vw"` 👍

**Review by jfplataroti — APPROVED** [review]
great use of grommet Card and CardFooter 👍 !

File: packages/web/components/cartItemSummary.tsx (hunk 1 above)
Comment (djoyal) [reply packages/web/components/cartItemSummary.tsx:154]: Thanks! I tried the predefined widths ('small' / 'medium' / etc...) but none matched the design.

**Pull request merged**